src/js/localstorage.js

```diff
@@ -0,0 +1,187 @@
+//TESTING LOCALSTORAGE WHETHER IT WORKS OR EXIST 
+export function localStorageTest() {                 //to avoid name collisions
+    
+    try {
+        localStorage.setItem("test","test");
+        localStorage.removeItem("test");
+        console.log('Local Storage Fine')
+        return true;
+    } 
+    catch(e) {
+        return false;
+    }
+}
+
+//LOCKING THE START BUTTON IF LOCAL STORAGE DOESNT WOKR
+export const lockStart = () => {
+    if (localStorageTest()) {
+      return;
+    } 
+    else {
+      document.querySelector("a").remove();
+      document.querySelector("main").insertAdjacentHTML("beforeend",'<button class="btn-fail">Start</button>');
+      document.querySelector(".btn-fail").addEventListener("mouseover", function () {
+      document.querySelector("main").insertAdjacentHTML("beforeend",'<p class="err-p">Fix your local storage</p>');});
+      document.querySelector(".btn-fail").addEventListener("mouseout", function () {document.querySelector(".err-p").remove();});
+    }
+  };
+
+//SAVING TO AND LOADING FROM LOCALSTORAGE - universal function
+
+export function saveToLocalStorage(resource, positionName) {
```

Comment:
Try to stick to one style of defining functions. Once you write `const lockStart = () => ...` then you use `function saveToLocalStorage(....) { ... }`

Reply:
Done.